vocab.py

```diff
@@ -1,18 +1,27 @@
+import click
 import json
 
 from PyPDF2 import PdfReader
 
-reader = PdfReader("a1_1.pdf")
-number_of_pages = len(reader.pages)
-page = reader.pages[0]
-text = page.extract_text()
 
-vocab = {}
-genders = {"der": "m", "die": "f", "das": "n"}
-for row in text.split("\n"):
-    row = row.split(" ")
-    if row[0] in ["der", "die", "das"] and row[1][0].isupper():
-        vocab.update({row[1]: {"gender": genders[row[0]]}})
+def read_pdf_to_json(file_path):
+    reader = PdfReader(file_path)
+    all_pages = len(reader.pages)
+    vocab = []
+    genders = {"der": "m", "die": "f", "das": "n"}
+    for i in range(all_pages):
+        page = reader.pages[i]
+        text = page.extract_text()
+        for row in text.split("\n"):
+            row = row.split(" ")
+            if row[0] in ["der", "die", "das"] and row[1][0].isupper():
+                vocab.append({"name": row[1], "gender": genders[row[0]]})
```

Comment:
Can we also please add another key value  `"id":"index"` in each object? 

The whole object would look like below:

`[ { "name":"Abend", "gender":"m", "id":"1" }, { "name":"Aussage", "gender":"f", "id":"2" } ]`